Plugins/Public/mobiledocking_plugin/Main.cpp

```diff
@@ -375,23 +465,6 @@ void __stdcall BaseEnter(uint iBaseID, uint client)
 		status += L"<TEXT>" + XMLText(mobiledockClients[client].wscDockedWithCharname) + L"</TEXT><PARA/><PARA/>";
 		status += L"<POP/></RDL>";
 		SendSetBaseInfoText2(client, status);
-
-		// Check to see that the carrier thinks this ship is docked to it.
-		// If it isn't then eject the ship to space.

```

Comment:
any idea how this was possible before? looks like IsShipDockedOnCarrier function is no longer in use

Reply:
Possible with help of SaveFiles.cpp.